app/mailers/patient_mailer.rb

```diff
@@ -175,12 +183,14 @@ def assessment_email(patient)
     @patients = patient.active_dependents.uniq.collect do |dependent|
       { patient: dependent, jurisdiction_unique_id: Jurisdiction.find_by_id(dependent.jurisdiction_id).unique_identifier }
     end
+
+    update_last_assessment_reminder_sent(patient) # Update last send attempt timestamp before Twilio call
```

Comment:
Comment says Twilio but this is for email.

Reply:
Fixed